pkg/sandbox/runx/impl/registry/extract.go

```diff
@@ -53,3 +55,23 @@ func contentDir(path string) string {
 	}
 	return filepath.Join(path, contents[0].Name())
 }
+
+func createSymbolicLink(src, dest, repoName string) error {
+	if err := fileutil.EnsureDir(dest); err != nil {
+		return err
+	}
+	if err := os.Chmod(src, 0755); err != nil {
+		return err
+	}
+	binaryName := filepath.Base(src)
+	// This is a good guess for the binary name. In the future we could allow
+	// user to customize.
+	if strings.Contains(binaryName, repoName) {
+		binaryName = repoName
+	}
+	err := os.Symlink(src, filepath.Join(dest, binaryName))
+	if errors.Is(err, os.ErrExist) {
+		return nil
+	}
```

Comment:
Might want to check that the symlink is pointing to the right binary if it already exists in case the user ever installs a different version.

Reply:
The symlink is already in a versioned directory: (Both the symlink and the source)

```
ls -al ~/Library/Caches/jetpack.io/pkgs/mvdan/gofumpt/v0.5.0/darwin/arm64/       
gofumpt -> /Users/mike/Library/Caches/jetpack.io/pkgs/mvdan/gofumpt/v0.5.0/gofumpt_v0.5.0_darwin_arm64
```

So nice to have in case we ever change the download logic but not urgent. Will add TODO